### DISCUSSION TITLE
Use system's borders setting

### DISCUSSION THREAD

**Discussion by Gemmstone** [zen-browser/desktop#2639] [Ideas]
Hi, this is kind of a petty ask-idea but, i'm using a riced KDE workspace and it would be great to have the option to disable the Client Side Decorations and change to Server Side, just so it fits with my current workflow.

I use things like highlited borders for focused windows and floating panels and the Client Side decorations get in the way of that, weirdly enough.

For comparison:

- firefox:
![image](https://github.com/user-attachments/assets/5f151317-84f1-4c5d-a77c-d46c7847a0f6)

- Zen:
![image](https://github.com/user-attachments/assets/0dbd429c-2579-43bd-b640-16810f5da2ba)

That's all!

**Comment by AndydeCleyre** [comment]
I'd been trying to find a way to enable regular (kwin) window decorations, to see if there's any chance I would use this browser, and you can! Right click a toolbar button, choose Customize Toolbar, and check the Title Bar box.

Maybe you're looking for a solution without the title bar though. If so, good luck.

IMO this should be one of the first-run getting started wizard choices.

**Reply by Gemmstone** [reply]
Yep! that was it! 

yeah and yeah, it should be on the wizard, i hate that it's so hidden, or at least detect that i'm using KDE and change it automagically lmao
